man/machinery-inspect-docker.1.md

```diff
@@ -0,0 +1,109 @@
+
+## inspect-container — Inspect Container
+
+### SYNOPSIS
+
+`machinery inspect-container` [OPTIONS] IMAGENAME
+
+`machinery inspect-container` [OPTIONS] IMAGEID
+
+`machinery` help inspect-container
+
+
+### DESCRIPTION
+
+The `inspect-container` command analysis a container image by creating, starting and inspecting a container from the provided image
+and generates a system description from the gathered data. After the inspection the container will be killed and removed again.
```

Comment:
I used `killed` because it an official command from docker we use. Stop does it officially a little bit different. Should I change it anyway ?